avionics/common/include/sensor_collection.h

```diff
@@ -90,8 +93,16 @@ class SensorCollection {
      */
     RocketStatus getStatus(bool refresh = false);
     const uint8_t *getStatusBitfield() const { return status_bitfield_; }
+
+    /**
+     * @brief update status of sensors to CRITICAL_FAILURE/NONCRITICAL_FAILURE if sensor experiences FAILURE
+     */
     void updateStatus();
 
+    /**
+     * @brief Get the sensor_data array from SensorCollection
```

Comment:
The exact naming may not always stay the same, so I think it'd be better to say something like 'Get the current array of sensor data'